common/api-review/performance-exp.api.md

```diff
@@ -7,16 +7,16 @@
 import { FirebaseApp } from '@firebase/app-exp';
 
 // @public
-export interface FirebasePerformance {
-    dataCollectionEnabled: boolean;
-    instrumentationEnabled: boolean;
-}
+export function getPerformance(app?: FirebaseApp): PerformanceMonitoring;
 
 // @public
-export function getPerformance(app?: FirebaseApp): FirebasePerformance;
+export function initializePerformance(app: FirebaseApp, settings?: PerformanceSettings): PerformanceMonitoring;
 
 // @public
-export function initializePerformance(app: FirebaseApp, settings?: PerformanceSettings): FirebasePerformance;
+export interface PerformanceMonitoring {

```

Comment:
Just curious about why we have this below. Wouldn't we want to "introduce it" before the function that uses it?

Reply:
This file is auto generated by api-extractor to describe APIs in human readable format. You can see that it's a md file, not actual code where the order also doesn't matter. Types are compile time artifacts, and are available to things before defined them in the file.

Reply:
BTW, I reverted Perf related changes because we decided to keep using `FirebasePerformance` as the name.

Reply:
Sounds good, thanks!